paf.opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "tls-mirage"
   "mimic"
   "cohttp-lwt"
-  "letsencrypt"
+  "letsencrypt" {>= "0.2.4"}
```

Comment:
this shouldn't be necessary since letsencrypt 0.2.4 did not change any API

Reply:
Yes, it's what I think when I wrote it, I will clean the history and delete it then 👍 thanks.